api/src/test/java/uk/gov/dhsc/htbhf/claimant/entity/converter/ListOfLocalDatesConverterTest.java

```diff
@@ -0,0 +1,82 @@
+package uk.gov.dhsc.htbhf.claimant.entity.converter;
+
+import com.fasterxml.jackson.core.JsonParseException;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.jupiter.api.Test;
+import org.mockito.ArgumentCaptor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+
+import java.io.IOException;
+import java.time.LocalDate;
+import java.util.List;
+
+import static java.util.Collections.singletonList;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.catchThrowableOfType;
+import static org.mockito.ArgumentMatchers.*;
+import static org.mockito.BDDMockito.given;
+import static org.mockito.Mockito.verify;
+
+@SpringBootTest
+class ListOfLocalDatesConverterTest {
+
+    @MockBean
+    private ObjectMapper objectMapper;
+
+    @Autowired
+    private ListOfLocalDatesConverter converter;
+
+    @Test
+    void shouldConvertJsonStringToListOfLocalDates() throws IOException {
+        String listJson = "[{\"2018-10-01\"}]";
+        List<LocalDate> dates = singletonList(LocalDate.now());
```

Comment:
As discussed, lets create a local date from the String used above in `listJson`

Reply:
Done.